src/main/webapp/script.js

```diff
@@ -11,15 +11,15 @@ if(username === "" || username === null){
         `<p>Hi ` + 
         username + 
         `!</p>` +
-        `<button class="btn create-btn" onclick="location.href='./pages/dashboard.html'">Dashboard</button>`;
+        `<button class="btn create-btn" onclick="location.href='./pages/dashboard.html'">Dashboard</button>
+        <button class="btn logout-btn" id="logout-btn">Log out</button>`;
 }
 
-// Direct user to sign up page
-document.getElementById("userSignup").onclick = () => {
-    location.href = "./pages/userSignup.html";
-}
-
-// Direct user to sign in page
-document.getElementById("userSignin").onclick = () => {
-    location.href = "./pages/userSignin.html";
+// Log out
+document.getElementById("logout-btn").onclick = function(){
+    sessionStorage.clear();
+    btnDiv.innerHTML = `<p color=red> Successully logged out</p>`
+    window.setTimeout(function() {
```

Comment:
nit: I think it's more common to do an alert("You've logged out") to inform users that they've logged out, and takes them back to index.html after they close the alert. An alert is more noticeable, but the message works as well.

Reply:
Good idea! I've edited it!